src/main/java/fr/esgi/al5_2/Tayarim/repositories/AdministrateurRepository.java

```diff
@@ -0,0 +1,35 @@
+package fr.esgi.al5_2.Tayarim.repositories;
+
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
+import lombok.NonNull;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+/**
+ * Interface de repository pour les opérations de base de données sur les entités Administrateur.
+ * Fournit des méthodes pour retrouver un administrateur par email ou par numéro de téléphone.
+ */
+@Repository
+public interface AdministrateurRepository extends JpaRepository<Administrateur, Long> {
+
+  /**
+   * Recherche le premier administrateur correspondant à l'adresse email spécifiée.
+   *
+   * @param email L'adresse email utilisée pour la recherche.
+   * @return Un Optional contenant l'administrateur trouvé ou un Optional vide si aucun
+   * administrateur n'est trouvé.
+   */
+  Optional<Administrateur> findFirstByEmail(@NonNull String email);
+
+  /**
+   * Recherche le premier administrateur correspondant au numéro de téléphone spécifié.
+   *
+   * @param numTel Le numéro de téléphone utilisé pour la recherche.
+   * @return Un Optional contenant l'administrateur trouvé ou un Optional vide si aucun
+   * administrateur n'est trouvé.
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocTagContinuationIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line continuation have incorrect indentation level, expected level should be 4.